pkg/plugin/server.go

```diff
@@ -43,7 +48,19 @@ func (s *KeyManagementServiceServer) Version(ctx context.Context, request *k8spb
 	}, nil
 }
 
+// Encrypt message
 func (s *KeyManagementServiceServer) Encrypt(ctx context.Context, request *k8spb.EncryptRequest) (*k8spb.EncryptResponse, error) {
+	start := time.Now()
+
+	var err error
+	defer func() {
+		if err != nil {
+			s.reporter.ReportRequest(ctx, metrics.EncryptOperationTypeValue, metrics.ErrorStatusTypeValue, time.Since(start).Seconds(), err.Error())
```

Comment:
The only diff in the `ReportRequest` call here is the status type value and errors. So could we just initialize a variable to set the `errorTypeValue = <value>` and call `s.reporter.ReportRequest` just once?

```go
errorStatus := metrics.SuccessStatusTypeValue
errs := ""
if err != nil {
    errorStatus = metrics.ErrorStatusTypeValue
    errs = err.Error()
}
s.reporter.ReportRequest(ctx, metrics.EncryptOperationTypeValue, errorStatus, time.Since(start).Seconds(), err.Error())
```

Reply:
refactored.